### ISSUE TITLE
Add file uploader

### ISSUE THREAD

**Issue by ericafenyo** [ericafenyo/bike-diary-api#18]
We want to keep a route that is responsible for uploading any type of file. As users explore and take pictures, we need to keep an archive of those pictures.

**Comment by ericafenyo** [comment]
I searched the nestjs docs for file uploads in the GraphQL category but couldn't find any. Finally, after several Google searches, I discovered an npm package `graphql-upload`.

**Comment by ericafenyo** [comment]
I was able to set up the basic file upload after some struggles, as `graphql-upload` version 13 and above deleted exports for files from their `index.js`. I ended up with imports like these:

```ts
import * as graphqlUploadExpress from 'graphql-upload/graphqlUploadExpress.js';
import * as GraphQLUpload from 'graphql-upload/GraphQLUpload.js';
```

Everything works now. Files can be uploaded using the `multipart/form-data`  content type.  However, one of the Apollo GraphQL blogs mentions that this approach is be vulnerable to a [CSRF mutation attack](https://github.com/apollographql/apollo-server/security/advisories/GHSA-2p3c-p3qw-69r4). But the most recent version (v3) of the Apollo server has built-in protection. 

We will have to study this approach and, if possible, use another implementation.
